build-system/compile/build.conf.js

```diff
@@ -76,26 +76,8 @@ function getReplacePlugin() {
     replacements.push(createReplacement(defineFlag, true));
   }
 
-  const currentTimestampMs = Date.now();
-
   // default each experiment flag constant to false
```

Comment:
Does this comment need to be updated too?

Reply:
I don't think so, still looks relevant.